app/components/Utils/TimeSince.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { connect } from 'react-redux'
-import differenceInSeconds from 'date-fns/difference_in_seconds'
-import distanceInWordsToNow from 'date-fns/distance_in_words_to_now'
+import differenceInSeconds from 'date-fns/differenceInSeconds'
+import distanceInWordsToNow from 'date-fns/formatDistance'
```

Comment:
According to https://date-fns.org/v2.16.1/docs/formatDistanceToNow, there is a `formatDistanceToNow` helper that matches the current function signature. I think we should use that, with the current code when I post a new comment I see `In less than one minute`:
![image](https://user-images.githubusercontent.com/1556356/103581736-d6e47480-4edc-11eb-9ae6-2bf12dbeab4e.png)

Reply:
Nice catch! It should be fixed by now :)

Reply:
I still see this issue on https://deploy-preview-728--captainfact.netlify.app/, do you get a different result locally?

Reply:
I can't seem to reproduce the issue on the preview deployment 😕 
The following screenshots are from [this video](https://deploy-preview-728--captainfact.netlify.app/videos/Jykl)

![image](https://user-images.githubusercontent.com/357835/104586984-00bc3a80-5645-11eb-90c3-8c7cf516ffb5.png)
![image](https://user-images.githubusercontent.com/357835/104587085-2b0df800-5645-11eb-8a20-0d258e6e16e7.png)
![image](https://user-images.githubusercontent.com/357835/104587097-32350600-5645-11eb-841e-d16efb7244bb.png)
![image](https://user-images.githubusercontent.com/357835/104587125-3a8d4100-5645-11eb-8e7e-17f7294b0597.png)

Reply:
It only appears when you post a new comment. I'm currently testing on latest Firefox:

![Peek 2021-01-20 19-31](https://user-images.githubusercontent.com/1556356/105218966-29b56180-5b56-11eb-8f3e-a47373cbb49f.gif)

Reply:
My bad, seems to be resolved in the latest version :+1: